src/azure-cli/azure/cli/command_modules/ams/tests/latest/recordings/test_ams_streaming_endpoint_update.yaml

```diff
@@ -1200,7 +1200,7 @@ interactions:
       accept-language:
       - en-US
     method: DELETE
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001?api-version=2019-07-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001?api-version=2018-05-01
```

Comment:
There may be something wrong in your recording file. Why does the api version used by resource groups change to 2018-08-01? It should be 2019-07-01.

Reply:
Resolved.